doc/nrf/app_dev/device_guides/nrf54h/index.rst

```diff
@@ -41,3 +41,4 @@ Zephyr and the |NCS| provide support and contain board definitions for developin
    ug_nrf54h20_debugging
    ug_nrf54h20_custom_pcb
    ug_nrf54h20_flpr
+   ../nrf54l/zms.rst
```

Comment:
```suggestion
   ../nrf54l/zms
```
don't think .rst is required here.

Reply:
the docbuild is green, and the location of that document is temporary.